src/com/company/Related.java

```diff
@@ -0,0 +1,34 @@
+package com.company;
+
+import java.util.Locale;
+import java.util.Scanner;
+
+public class Related2 {
+    public static boolean Realted(String first, String second) {
+
+
+        if(first.length()!=second.length())
+            return false;
+        for(int i=0;i<first.length();i++){
+            for(int j=0;j<second.length();j++){
+                if(first.charAt(i)==second.charAt(j)){
+                    second = second.substring(0, j) + second.substring(j + 1);
+                    break;
+                }
```

Comment:
This has two nested loops, so for this the complexity is `O(n^2)`. You can do this in 1 loop and `O(n)` complexity.